src/components/Menu.jsx

```diff
@@ -1,29 +1,75 @@
 import React, { useState, useEffect } from "react";
-import { Segment, List } from "semantic-ui-react";
+import { Segment, Grid, Menu, Container } from "semantic-ui-react";
 import MenuItem from "./MenuItem";
 import axios from "axios";
 
-const Menu = () => {
+const MenuPage = () => {
   const [menuItems, setMenuItems] = useState([]);
+  const [activeCategory, setActiveCategory] = useState("starters");
+
   useEffect(() => {
     axios.get("https://slowfood.heroku.com/api/products").then((response) => {
       setMenuItems(response.data.products);
     });
   }, []);
+  useEffect(() => {
+    axios.get("https://slowfood.heroku.com/api/products").then((response) => {
+      setMenuItems(response.data.products);
+    });
+  }, [activeCategory]);
+
+  const filteredCategory = menuItems.filter(
```

Comment:
I don't like the naming, filteredCategory doesn't really say what that variable contains. Maybe itemsInCategory?

Reply:
Updated